### DISCUSSION TITLE
Copilot issues with vscode

### DISCUSSION THREAD

**Discussion by arnoldasr** [community/community#40342] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/87844946/204508044-f143ae4f-c18d-44d5-9819-3dbbcd7b9371.png)
![image](https://user-images.githubusercontent.com/87844946/204508097-8311fdd0-f001-499d-8e58-0627ec39f24c.png)

As per the screenshots, it stated that I am eligible for copilot for free but in vscode it says I need a subscription 

Could you please help me with how I can solve this? Thank you in advance

**Comment by Akash1134** [comment, marked as answer]
Hi @arnoldasr  The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) 

Also, please note that over the weekend we deployed a new update that didn't work as expected and affected some of the Copilot subscribers and they have reported a similar problem. Our support team will be able to look up your subscription details and diagnose the issue accordingly. Thanks!

**Reply by arnoldasr** [reply]
Thank you! Have a nice day
